src/test/java/ru/hh/jetty/HHServerConnectorFailFastTest.java

```
@@ -0,0 +1,139 @@
+package ru.hh.jetty;
+
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.util.thread.QueuedThreadPool;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Test;
+
+import java.io.EOFException;
+import java.net.Socket;
+import java.net.SocketException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static ru.hh.jetty.HHServerConnectorTestUtils.createServer;
+import static ru.hh.jetty.HHServerConnectorTestUtils.getPort;
+import static ru.hh.jetty.HHServerConnectorTestUtils.repeat;
+
+public class HHServerConnectorFailFastTest {
```

Новый тест. На fail fast.